.changeset/chubby-olives-say.md

```diff
@@ -0,0 +1,25 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": tree
+---
+
+New Alpha APIs for tree data import and export
+
+A collection of new `@alpha` APIs for important and exporting tree content and schema from SharedTrees has been added to `TreeAlpha`.
+These include import and export APIs for for `VerboseTree`, `ConciseTree` and compressed tree formats.
+
+`TreeAlpha.create` is also added to allow constructing trees with a more general API than having to use the schema constructor directly (since that doesn't handle polymorphic roots, or non-schema aware code).
```

Comment:
```suggestion
`TreeAlpha.create` is also added to allow constructing trees with a more general API instead of having to use the schema constructor directly (since that doesn't handle polymorphic roots, or non-schema aware code).
```